autoload/chex.wad/brghtmps.lmp

```diff
@@ -0,0 +1,81 @@
+BRIGHTMAP NOTGRAY 4,9-79,112-255
+BRIGHTMAP GREENONLY1 112-127
+BRIGHTMAP GREENONLY2 112-125
+BRIGHTMAP GREENONLY3 112-123
+BRIGHTMAP REDANDGREEN 16-47,112-127,173-191
+BRIGHTMAP CHEXRED 176-177
+BRIGHTMAP CHEXGREEN 112-115
+BRIGHTMAP CHEXREDGREEN 45,112-118,174-177
+
+TEXTURE BIGDOOR1 GREENONLY3
+// {"BIGDOOR4", DOOM1AND2, greenonly3}, // C1: some stray green pixels, C2: many stray green pixels
```

Comment:
I'd prefer if these were converted from C syntax to the actual BRGHTMPS lump format, so that in theory it would be sufficient to just uncomment these lines.

Reply:
I'm not sure about Chex 2, is it Doom 2 or Doom 1? I will check later.

Reply:
Chex 2 is a PWAD to Chex 1, i.e. Doom 1.

Reply:
However, in the context of brightmap parsing, calling Chex 2 as Doom 2 to differentiate it from Chex 1 is unambigious.

Reply:
Should we make `chex-all`, `chex.wad` and `chex2.wad` folders? 😄

Reply:
We could. But, since chex2.wad is *always* loaded after chex.wad, its BRGHTMPS entries should override the ones for chex.wad anyway, right? Or, we could introduce a `CHEX1|CHEX2` field in the BRGHTMPS spec if you don't like misusing the `DOOM1|DOOM2` namespace. 😉